Wrappers.h

```diff
@@ -1,50 +1,43 @@
+#pragma once
+#include "Compressor.h"
+#include "Bakery.h"
+#include "House.h"
+#include "CompressorRack.h"
+#include "Neighborhood.h"
+
```

Comment:
Remember, this chapter was called "Don't Include what you don't use."
You aren't using any of these classes inside this header file.
"using" means accessing member variables or member functions.
The compiler already knows how large pointers are.
It has no issue when you declare a pointer to a forward-declared type.
- [ ] remove these include directives.  Forward declare these types.  Include in the cpp file
- [ ] revise